enterprise/api/v1/serializers.py

```diff
@@ -914,3 +916,46 @@ def validate(self, data):  # pylint: disable=arguments-differ
             )
 
         return data
+
+
+class Base64EmailCSVField(serializers.Field):
+    """
+    Serializers a Base64 encoded CSV with emails into an array of emails
+    """
+    def to_internal_value(self, data):
+        decoded_email_csv = base64.standard_b64decode(data).decode('utf-8').splitlines()
+        email_csv = list(csv.DictReader(decoded_email_csv))
+        if email_csv and 'email' not in email_csv[0]:
+            raise serializers.ValidationError(
+                "The .csv file must have a column of email addresses, "
+                "indicated by the heading 'email' in the first row."
+            )
+        return [email.get('email') for email in email_csv]
+
+    def to_representation(self, value):
+        return value
+
+
+# pylint: disable=abstract-method
+class EnterpriseCustomerBulkEnrollmentsSerializer(serializers.Serializer):
+    email = serializers.CharField(required=False)
+    email_csv = Base64EmailCSVField(required=False)
+    course_run_key = serializers.CharField(required=False)
```

Comment:
It seems like this is sort of required, why not make the serializer require it?

Reply:
It's not required today which'll result in just linking the learners to an enterprise. Keeping parity with how the admin form works today here.
```    
        course = forms.CharField(
        label=_("Enroll these learners in this course"), required=False,
        help_text=_("To enroll learners in a course, enter a course ID."),
    )```